apps/dashboard/src/components/conditions-editor/variable-select.tsx

```diff
@@ -11,6 +11,7 @@ type VariableSelectProps = Omit<HTMLAttributes<HTMLDivElement>, 'onChange'> & {
   value?: string;
   options: Array<{ label: string; value: string }>;
   onChange: (value: string) => void;
+  onInputChange?: (value: string) => void;
```

Comment:
allow listening on input change; the `onChange` is called when the item is picked from the dropdown